cloudformation.yaml

```diff
@@ -493,37 +485,16 @@ Resources:
     - NodeRotationStepFunction
     - TriggerExecutionRole
 
-  MasterNodeRotationSchedule:
-    Condition: HasDedicatedMasters
-    Type: AWS::Events::Rule
-    Properties:
-      Name: !Sub ${Stack}-master-node-rotation-schedule-${Stage}
-      ScheduleExpression: !Ref MasterNodeRotationCronExpression
-      State: ENABLED
-      Targets:
-      - Arn: !Ref NodeRotationStepFunction
-        RoleArn: !GetAtt TriggerExecutionRole.Arn
-        Id: !GetAtt NodeRotationStepFunction.Name
-        Input: !Sub
-          - |
-            {
-              "asgName": "${DedicatedMasterAsg}",
-              "stepFunctionArn": "${StepFunctionArn}"
-            }
-          -
-            StepFunctionArn: !Ref NodeRotationStepFunction
-    DependsOn:
-    - NodeRotationStepFunction
-    - TriggerExecutionRole
-
   ExecutionFailureAlarm:
     Type: AWS::CloudWatch::Alarm
     Properties:
       AlarmActions:
       - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${SNSTopicForAlerts}
-      AlarmName: !Sub Failed to complete node rotation for ${Stack}, ${Stage} Elasticsearch Cluster
+      AlarmName: !Sub Failed to complete node rotation for an elasticsearch cluster
       AlarmDescription: !Sub
-        - Elasticsearch Node Rotation failed - please see Step Function execution history for ${StepFunctionName}
+        - >
+          Elasticsearch Node Rotation failed - please see Step Function execution history for ${StepFunctionName}.
+          You can find the affected auto scaling group by clicking on the AutoScalingGroupCheck node in the execution history and clicking "Step Input"
```

Comment:
We don't now know which cluster or which autoscaling groups we are modifying up front so unfortunately we can't include that in the alarm. @amyhughes and I discussed this and decided it would probably be enough to add instructions on how to find out which one was affected by looking in the Step Functions console. In her experience on Ophan, which has all three types of autoscaling group, we had to look at the output anyway to diagnose what went wrong.

A hacky option to re-introduce this could be to manually write to a custom CloudWatch metric on failure with the autoscaling group as a dimension. This gets a bit awful in the state machine though as we wouldn't want to add a `Catch` to each step. That said, you can add a `Catch` to a `Map` step, so if we re-introduced the functionality to operate on multiple autoscaling groups (removed from this PR for simplicity) we could add the catch to the `Map` step there.

Reply:
> In her experience on Ophan, which has all three types of autoscaling group, we had to look at the output anyway to diagnose what went wrong.

I've always needed to do this when debugging failures which have occurred against Central ELK too, so the suggestion here sounds good to me.